azure-kusto-ingest/azure/kusto/ingest/exceptions.py

```diff
@@ -44,3 +44,15 @@ class KustoMissingMappingReferenceError(KustoClientError):
     def __init__(self):
         message = "When stream format is json, mapping name must be provided."
         super(KustoMissingMappingReferenceError, self).__init__(message)
+
+
+class KustoInvalidEndpointError(KustoClientError):
```

Comment:
This has a side effect where anyone who used to catch kusto client exceptions will now crash

Reply:
Are you sure? it inherits KustoClientError..

Reply:
Hmm... you may have a point :)